Source/Urho3D/AngelScript/MathAPI.cpp

```diff
@@ -218,17 +218,17 @@ static void RegisterIntVector2(asIScriptEngine* engine)
     engine->RegisterObjectMethod("IntVector2", "IntVector2& opAddAssign(const IntVector2&in)", asMETHOD(IntVector2, operator +=), asCALL_THISCALL);
     engine->RegisterObjectMethod("IntVector2", "IntVector2& opSubAssign(const IntVector2&in)", asMETHOD(IntVector2, operator -=), asCALL_THISCALL);
     engine->RegisterObjectMethod("IntVector2", "IntVector2& opMulAssign(const IntVector2&in)", asMETHODPR(IntVector2, operator *=, (const IntVector2&), IntVector2&), asCALL_THISCALL);
-    engine->RegisterObjectMethod("IntVector2", "IntVector2& opMulAssign(int)", asMETHODPR(IntVector2, operator *=, (int), IntVector2&), asCALL_THISCALL);

```

Comment:
1. float -> int conversion is losing. Try IntVector2(1, 1) * 2kkk before and after the commit.
2. Legacy code will spam warnings after this change

Reply:
Yup already saw it on CI and pushed a commit addressing it. Left scalar multiplication and division operators to always take `float` as parameter.

Reply:
> Left scalar multiplication and division operators to always take float as parameter

I meant, use float for scalar ops with IntVector2/3 is bad too.